tests/baselines/reference/docker/test262.log

```diff
@@ -0,0 +1,11028 @@
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.1.1_A2.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.1.1_A2.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.2.1_A1.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.2.1_A1.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.2.1_A2.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.2.1_A2.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.2.1_A3.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.2.1_A3.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.2.1_A4.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.2.1_A4.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.3_A1.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.3_A1.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.3_A2.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.3_A2.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.3_A3.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.3_A3.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S9.2_A1_T1.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S9.2_A1_T1.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S9.2_A2_T1.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S9.2_A2_T1.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S9.2_A3_T1.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S9.2_A3_T1.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S9.2_A4_T1.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S9.2_A4_T1.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S9.2_A4_T3.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S9.2_A4_T3.js
```

Comment:
These tests use extended unicode escapes, which we only even parse when `target` is es6 or above. So we parse this test as garbage and emit corresponding garbage.